src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/NonEmptyAtclauseDescriptionCheck.java

```diff
@@ -67,44 +67,158 @@
  * <pre>
  * class Test
  * {
- * &#47;**
- * * Violation for param "b" and at tags "deprecated", "throws".
- * * &#64;param a Some javadoc // OK
- * * &#64;param b
- * * &#64;deprecated
- * * &#64;throws Exception
- * *&#47;
- * public int method(String a, int b) throws Exception
+ *  // Violation for param "b" and at tags "deprecated", "throws" and "return".
+ *  &#47;**
+ *  * Some summary.
+ *  * &#64;param a Some description
+ *  * &#64;param b
+ *  * &#64;deprecated
+ *  * &#64;throws Exception
+ *  * &#64;return
+ *  *&#47;
+ * public int method(String a, int b) throws Exception {
+ *   return 1;
+ *  }
+ * }
+ * </pre>
+ * <p>
+ * To configure the check to validate {@code @param} tags:
+ * </p>
+ * <pre>
+ * &lt;module name="NonEmptyAtclauseDescription"&gt;
+ *   &lt;property name="javadocTokens" value="PARAM_LITERAL"/&gt;
+ *  &lt;/module&gt;
+ * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * class Test
  * {
- * return 1;
+ *  // Violation 4 lines below on param tag.
+ *  &#47;**
+ *  * Some summary.
+ *  * &#64;param a Some description
+ *  * &#64;param b
+ *  * &#64;deprecated
+ *  * &#64;throws Exception
+ *  * &#64;return
+ *  *&#47;
+ * public int method(String a, int b) throws Exception {
+ *   return 1;
+ *  }
  * }
+ * </pre>
+ * <p>
+ * To configure the check to validate {@code @return} tags:
+ * </p>
+ * <pre>
+ * &lt;module name="NonEmptyAtclauseDescription"&gt;
+ *   &lt;property name="javadocTokens" value="RETURN_LITERAL"/&gt;
```

Comment:
Make this config for param and expection tags. 

Remove all other example below this example. We do not need them. We just need to show users how to do customization on few tags , to show that coma is separator.

Reply:
@romani We need to keep the examples only for the default check, and only for the @param and @exception tags, and remove all other examples. Is that what you are saying?

Reply:
Yes